medical_practitioner/models/medical_role.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# Copyright 2017 Creu Blanca
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models, fields
+
+
+class MedicalRole(models.Model):
```

Comment:
Can you add `active = fields.Boolean('Active', default=True)` and show this field in the form view?

That way users can deactivate roles that have become obsolete.